operations.py

```diff
@@ -0,0 +1,292 @@
+import base64
+
+import requests
+from connectors.core.connector import get_logger, ConnectorError
+from requests.auth import HTTPBasicAuth
+
+logger = get_logger('qualyswas')
+
+
+class QualysWAS:
+    def __init__(self, config):
+        self.base_url = config.get('server_url').strip('/') + '/qps/rest/3.0/'
+        if not self.base_url.startswith('https://'):
+            self.base_url = 'https://{0}'.format(self.base_url)
+        self.username = config['username']
+        self.password = config['password']
+        self.verify_ssl = config['verify_ssl']
+        self.error_msg = {
+            400: 'The parameters are invalid.',
+            401: 'Invalid credentials were provided',
+            403: 'Access Denied',
+            404: 'The requested resource was not found',
+            409: 'The requested settings conflict with the current settings',
+            410: 'Cannot find the specified object',
+            422: 'Unable to process the request because system lockdown is currently disabled, or some file fingerprint lists or file names were already assigned',
+            423: 'The resource to update is locked and cannot be updated',
+            500: 'Internal Server Error',
+            503: 'Service Unavailable',
+            'time_out': 'The request timed out while trying to connect to the remote server',
+            'ssl_error': 'SSL certificate validation failed'}
```

Comment:
instead of custom error messages, pass the error message from api.

Reply:
done